.make/go/test.make

```diff
@@ -14,8 +14,8 @@
 
 # run tests
 go.test: go.min-version key.pem cert.pem
-	@$(log) "Testing `$(TEST_PACKAGES) | $(count)` go packages"
-	$(GO) test $(GO_FLAGS) $(GO_TEST_FLAGS) `$(TEST_PACKAGES)`
+	@$(log) "Running tests..."
+	$(GO) test $(GO_FLAGS) $(GO_TEST_FLAGS) ./...
```

Comment:
https://golang.org/doc/go1.9#vendor-dotdotdot
We require `1.11` for our builds anyway, so no reason be compatible with pre-`1.9`
Besides, by using `./...` we also ensure that packages, which do not contain tests and are not imported by tested packages actually compile(e.g. `cmd`) and since we don't do `build-all` on PRs anymore, we need to test that e.g. CLI can be built.